### PR TITLE
Fix memorial homepage

### PR DESCRIPTION
**Pull request by chrisrhymes**
- Fix memorial homepage icons
- Add story to show Memorial homepage with isBoxed
- Check if the serviceLink has quickLinks before mapping over them

### CODE REVIEW

File: src/library/components/icons/services/CondolenceBookIconHover/Icon.tsx [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-        </svg>
-    )
-} 
+  const randomId = Date.now().toString(36) + Math.random().toString(36).substr(2);
 
+  return (
+    <svg width="76" height="76" viewBox="0 0 76 76" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        fillRule="evenodd"
```
Comment (chrisrhymes) [inline src/library/components/icons/services/CondolenceBookIconHover/Icon.tsx:11]: fill-rule and clip-rule updated to fillRule and clipRule, the rest is prettier

File: src/library/components/icons/services/RoyalWebsiteIconHover/Icon.tsx [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-    export default ServiceIcon;
-    
-    
\ No newline at end of file
+  const randomId = Date.now().toString(36) + Math.random().toString(36).substr(2);
+
+  return (
+    <svg width="76" height="76" viewBox="0 0 76 76" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        fillRule="evenodd"
```
Comment (chrisrhymes) [inline src/library/components/icons/services/RoyalWebsiteIconHover/Icon.tsx:11]: As per previous icon

File: src/library/pages/MemorialPage/MemorialPage.stories.tsx [hunk 3]
```diff
@@ -366,28 +369,34 @@ ExampleHome.args = {
...
-    }
-  ]
-}
+      title: 'Newsletter',
+      url: '/',
+    },
+  ],
+};
+
+export const ExampleHomeBoxed = Template.bind({});
```
Comment (chrisrhymes) [inline src/library/pages/MemorialPage/MemorialPage.stories.tsx:398]: New story added, the rest is prettier

File: src/library/pages/MemorialPage/MemorialPage.tsx [hunk 4]
```diff
@@ -1,123 +1,146 @@
...
+        </PageStructures.PageWrapper>
 
-    <PageStructures.MaxWidthContainer>
-        <PageStructures.PageMain>
-          <PageStructures.ServicesLinksList 
-            serviceLinksArray={servicesArray}
-          />
+        <PageStructures.MaxWidthContainer>
+          <PageStructures.PageMain>
+            <PageStructures.ServicesLinksList serviceLinksArray={servicesArray} isBoxed={isBoxed} />
```
Comment (chrisrhymes) [inline src/library/pages/MemorialPage/MemorialPage.tsx:123]: Added isBoxed

File: src/library/structure/ServicesLinksBoxed/ServicesLinksBoxed.tsx [hunk 5]
```diff
@@ -43,7 +43,7 @@ const ServiceLinksBoxed: React.FunctionComponent<ServiceLinksBoxedProps> = ({ se
             <Styles.QuickLinksContainer show={index === activeServiceLink} ref={refs.current[index]}>
               <Styles.QuickLinksInnerContainer>
                 <Row>
-                  {serviceLink.quickLinksArray.map((quickLink, quickLinkIndex) => (
+                  {serviceLink.quickLinksArray?.map((quickLink, quickLinkIndex) => (
```
Comment (chrisrhymes) [inline src/library/structure/ServicesLinksBoxed/ServicesLinksBoxed.tsx:46]: Check there are quickLinksArray

**Review by DamienBurkeWN — CHANGES_REQUESTED** [review]
Works in Memorial page example there are 4 violations in the accessibility tab; on MemorialHero component there is 1 (duplicate ID)... these look minor but will no doubt get bounced back to us at some point if not sorted now.

File: src/library/structure/MemorialHero/MemorialHero.styles.js [hunk 6]
```diff
@@ -1,14 +1,13 @@
-import styled from "styled-components";
+import styled from 'styled-components';
 
-
-export const Wrapper = styled.header`
-background: ${props => props.theme.theme_vars.colours.black};
-overflow: ${props => props.noPadding ? "hidden" : "visible"};
+export const Wrapper = styled.section`
```
Comment (chrisrhymes) [inline src/library/structure/MemorialHero/MemorialHero.styles.js:3]: Prevent duplicate <header> so using section and added an aria-label

**Review by DamienBurkeWN — APPROVED** [review]
👍

**Pull request merged**